terraform/global-resources/aws_guardduty/event-pattern.json

```diff
@@ -0,0 +1,34 @@
+{
+  "detail-type": [
+    "GuardDuty Finding"
+  ],
+  "source": [
+    "aws.guardduty"
+  ],
+    "detail": {
+      "severity": [
+        7,
+        7.0,
+        7.1,
+        7.2,
+        7.3,
+        7.4,
+        7.5,
+        7.6,
+        7.7,
+        7.8,
+        7.9,
+        8,
+        8.0,
+        8.1,
+        8.2,
+        8.3,
+        8.4,
+        8.5,
+        8.6,
+        8.7,
+        8.8,
+        8.9
+      ]
+    }
+  }
```

Comment:
As discussed, create a dir here `/terraform/global-resources/resources` and store additional files.